components/metrics/DEPS

```diff
@@ -0,0 +1,19 @@
+# This component is shared with the Chrome OS build, so it's important to limit
```

Comment:
Do we care about the references to Chrome OS here and in README? Or do we just bring everything in wholesale?

Reply:
We don't use DEPS files for anything ( `depot_tools` construct and we deprecated DT ). Favor making minimal modifications to upstream code, so leaving it unchanged is correct.